examples/async-user-confirmation/genkit/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import "dotenv/config";
 
 import Enquirer from "enquirer";
-import { genkit } from "genkit";
+import { genkit } from "genkit/beta";
```

Comment:
all `session` functions are `beta`, so we need to instantiate `genkit` on examples with the `beta` constructor